core/src/com/unciv/logic/city/CityStats.kt

```diff
@@ -108,19 +108,7 @@ class CityStats {
     }
 
     private fun getStatPercentBonusesFromDifficulty(): Stats {
```

Comment:
If the function is no longer relevant, delete it

Reply:
Done.